core/storage/question/gae_models.py

```diff
@@ -577,7 +627,10 @@ def get_offset(query):
         return question_skill_link_models
 
     @classmethod
-    def get_all_question_ids_linked_to_skill_id(cls, skill_id):
+    def get_all_question_ids_linked_to_skill_id(
+            cls,
+            skill_id: str
```

Comment:
I think we can do
```suggestion
            cls, skill_id: str
```

(if all the args fit on one line it is fine to keep multiple on one line)

Reply:
Done on all the similar places